packages/reports/addon/templates/components/metric-selector.hbs

```diff
@@ -43,18 +43,10 @@
               {{on "click" (fn @onToggleMetricFilter metric)}}
             >
               <EmberTooltip @side="right" @popperContainer="body" @effect="none">
-                {{#if (has-parameters metric)}}
-                  {{#if (has-unfiltered-parameters metric @request)}}
-                    Add next filter
-                  {{else}}
-                    Remove all
-                  {{/if}}
+                {{#if (get-shallow this.metricsFiltered metric.id)}}
+                  Remove Filter
```

Comment:
With parameterized columns, I wonder if this still makes sense

Reply:
Yea I think at some point we talked about moving away from the toggle filter setup to just adding a new filter. It's also in a slightly weird state now since you can't change parameters from filters so you likely need to add/remove the filter from the column config